addons/account_edi/views/account_edi_document_views.xml

```diff
@@ -1,19 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Need to do upgrade branch
`util.remove_view(cr, 'account_edi.view_tree_account_edi_document')`